WORKSPACE

```diff
@@ -84,11 +84,19 @@ filegroup(
 load("@io_bazel_rules_scala//scala:toolchains.bzl","scala_register_toolchains")
 scala_register_toolchains()
 
-load("//scala:scala_maven_import_external.bzl", "scala_maven_import_external")
+load("//scala:scala_maven_import_external.bzl", "scala_maven_import_external", "java_import_external")
 scala_maven_import_external(
     name = "com_google_guava_guava_21_0",
     licenses = ["notice"],  # Apache 2.0
     artifact = "com.google.guava:guava:21.0",
     server_urls = ["https://mirror.bazel.build/repo1.maven.org/maven2"],
     jar_sha256 = "972139718abc8a4893fa78cba8cf7b2c903f35c97aaf44fa3031b0669948b480",
+)
+
+# for regression testing purposes
```

Comment:
i don't understand this (even with the comment sorry!)

Reply:
changed a more detailed explanation. is this clear now?